src/app/analyze/stack/stack-details/render-component.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/operator/map';
+
+import { StackAnalysesModel } from '../stack-analyses.model';
+
+@Injectable()
+export class RenderComponentService {
+
+  private componentAnalysesUrl = process.env.STACK_API_URL + '/analyses';
+
+  constructor(private http: Http, private stackAnalysesModel: StackAnalysesModel) { }
+
+
+  getComponentAnalyses(data: StackAnalysesModel): Observable<any> {
```

Comment:
It appears this function does not get called and the file "component-analyses.service.ts" is the same file with minor differences. Please delete one of the files OR make this different enough and "called" so there is a reason to keep it.